packages/core/src/components/footer/test/default/footer.e2e.ts

```diff
@@ -0,0 +1,39 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/footer/test/default/index.html';
+
+test.describe('tds-footer-default', () => {
+  test('renders default footer correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const footerComponent = page.locator('footer');
+    await expect(footerComponent).toHaveCount(1);
+
+    /** Check screenshot diff */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('Footer contains four links "Link text"', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const footerLinks = page.getByText('Link text');
+    await expect(footerLinks).toHaveCount(4);
+  });
+
+  test('Footer contains three truck icons', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const footerTruckIcons = page.getByRole('img');
+    await expect(footerTruckIcons).toHaveCount(3);
```

Comment:
The page.getByRole('img'); locator could also find the Scania logo (if the implementation was different) and then I believe this test would fail.

Reply:
hmm not sure I follow, might need a talk about this because if eels like you have been thinking about something that I don't follow fully on.

Reply:
I guess he means that the test is brittle in the sense that if the Scania logo was implemented as a image, the test would fail?

Reply:
Yes, i mean that then the getByRole('img') would find two elements, i believe

Reply:
Hmm ok, I'll rethink the locator.

Reply:
How about these? @alexandereneroth @timrombergjakobsson [dd8b96e](https://github.com/scania-digital-design-system/tegel/pull/462/commits/dd8b96ea1df9b59e5e063702a740de430255af69)

Reply:
I think this works OK. Nice workaround :)